docs/versioned_docs/version-7.0.x/configuration/overview.md

```diff
@@ -78,6 +78,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--oidc-jwks-url` | string | OIDC JWKS URI for token verification; required if OIDC discovery is disabled | |
 | `--oidc-email-claim` | string | which OIDC claim contains the user's email | `"email"` |
 | `--oidc-groups-claim` | string | which OIDC claim contains the user groups | `"groups"` |
+| `--oidc-claims-parameter` | string | additional OIDC claims parameter | |
```

Comment:
This feature won't be in the older versions so we need don't to add it to the older documentation

Reply:
Removed.